x-pack/plugins/security/public/management/management_service.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Subscription } from 'rxjs';
+import { i18n } from '@kbn/i18n';
+import { CoreSetup, FatalErrorsSetup } from 'src/core/public';
+import {
+  ManagementApp,
+  ManagementSetup,
+  ManagementStart,
+} from '../../../../../src/plugins/management/public';
+import { SecurityLicense } from '../../common/licensing';
+import { AuthenticationServiceSetup } from '../authentication';
+import { PluginStartDependencies } from '../plugin';
+import { APIKeysManagementApp } from './api_keys';
+import { RoleMappingsManagementApp } from './role_mappings';
+import { RolesManagementApp } from './roles';
+import { UsersManagementApp } from './users';
+
+interface SetupParams {
+  management: ManagementSetup;
+  license: SecurityLicense;
+  authc: AuthenticationServiceSetup;
+  fatalErrors: FatalErrorsSetup;
+  getStartServices: CoreSetup<PluginStartDependencies>['getStartServices'];
+}
+
+interface StartParams {
+  management: ManagementStart;
+}
+
+export class ManagementService {
+  private license!: SecurityLicense;
+  private licenseFeaturesSubscription?: Subscription;
+
+  setup({ getStartServices, management, authc, license, fatalErrors }: SetupParams) {
+    this.license = license;
+
+    const securitySection = management.sections.register({
+      id: 'security',
+      title: i18n.translate('xpack.security.management.securityTitle', {
+        defaultMessage: 'Security',
+      }),
+      order: 100,
```

Comment:
__note:__ Just noticed that [ML also uses 100](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/ml/public/application/management/index.ts#L29) :see_no_evil: